src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -216,14 +314,37 @@ public IsNullFilter(String path) {
    */
   public static class ExistsFilter extends SetFilterBase<String> {
     public ExistsFilter(String path, boolean existFlag) {
-      super("exist_keys", path, Operator.CONTAINS);
+      super("exist_keys", path, path, Operator.CONTAINS);
+    }
+
+    @Override
+    JsonNode asJson(JsonNodeFactory nodeFactory) {
+      return null;
```

Comment:
Should this be `nodeFactory.nullNode()` instead? Or are we not inserting explicit `null`s?

Would probably be good to test against Mongo implementation; I think (but could be wrong) that Mongo tends to add explicit `null`s.

Reply:
For values that need to be added as null to the document I have returned nodeFactory.nullNode(). The filters for which we shouldn't be adding data I have this method return null.

Reply:
Ok; logic for returning/receiving `null` should probably be explained in Javadocs of `asJson()`.
Although isn't there some overlap between this and `canAddFied()`? (as in either one can prevent addition, I think)